ci/build.yml

```
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.21.3
+    tag: 1.21.3-bullseye
```

[Newer version is available](https://hub.docker.com/_/golang)
```suggestion
    tag: 1.21.5-bullseye
```